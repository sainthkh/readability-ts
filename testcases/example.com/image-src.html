<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Title Name</title>
</head>

<body>
    <article>
<p><img src="image/chunks-of-bytecode/phases.png" alt="Phases of the two
implementations. jlox is Parser to Syntax Trees to Interpreter. clox is Compiler
to Bytecode to Virtual Machine." /></p>
    </article>

</body>

</html>