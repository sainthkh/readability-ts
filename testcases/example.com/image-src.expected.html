
<p><p>!$[[0 Reading/Title Name/images/image/chunks-of-bytecode/phases.png]]$</p></p>
    